js/scripts.js

```diff
@@ -5,7 +5,7 @@ function Telefon(marka, cena, kolor, klawiatura) {
 	this.klawiatura = klawiatura;
 }
 Telefon.prototype.printInfo = function() {
```

Comment:
please don't polish language in your code :) polish lang can be use only for translations ;)